solr/core/src/java/org/apache/solr/cloud/api/collections/CreateCollectionCmd.java

```diff
@@ -321,6 +319,8 @@ public void call(ClusterState clusterState, ZkNodeProps message, @SuppressWarnin
         }
       }
       if(isPRS) {
```

Comment:
@noblepaul do we need `prs` check here?

Reply:
yes, 
The idea is , we write the entire modified state.json in one command if it is PRS

Reply:
1. can be done same for non-prs?
2. lint 324 "ocmh.overseer.submit(new RefreshCollectionMessage(collectionName));" it should be same for prs and non-prs?